src/components/Dropdown/index.jsx

```diff
@@ -0,0 +1,24 @@
+import DeletedButton from "../DeletedButton"
+import InboxButton from "../InboxButton"
+import NewEmailButton from "../NewEmailButton"
+import SentButton from "../SentButton"
+
+function Dropdown (){
+    return (
+        <div className="dropDownContent">
+            <div className="grid grid-row-4 font-bold text-xl float-left
+            text-white bg-gray-400 absolute">
+                <button className="border-b
+                 hover:bg-slate-500 p-5"><NewEmailButton /></button>
+                <button className="border-b
+                 hover:bg-slate-500 p-5"><InboxButton /></button>
+                <button className="border-b
+                 hover:bg-slate-500 p-5"><SentButton /></button>
+                <button className="border-b
+                 hover:bg-slate-500 p-5"><DeletedButton /></button>
```

Comment:
So a few issues here. 

Your individual components (NewEmailButton, InboxButton, SentButton and DeletedButton) aren't really all that useful since they just contain a div and some text. 

Instead, you should create one single component for a nav link which contains the button with it's tailwind classes, and then accepts the text to display as a prop.

So this component would end up displaying that component for each link.

Also - as part of story 1, please only display the inbox button - none of the others work yet so we don't want them in main.

And in addition to that, the inbox link doesn't actually do anything - it needs to link to the homepage.